README.rst

```diff
@@ -52,16 +52,17 @@ been imported via:
 
 .. code::
 
-    from stdio_mgr import stdio_mgr
+    >>> from stdio_mgr import stdio_mgr
 
 **Mock** ``stdout``\ **:**
 
 .. code::
 
+    >>> import os
```

Comment:
I like to put imports like this into the doctest setup, to streamline the code examples.

Pls move this into [here](https://github.com/bskinn/stdio-mgr/blob/cef7a932a37ffe0f4b01cc267ba77885b4f6435e/conftest.py#L65-L68).

Reply:
That doesnt appear to work in some instances, as I needed to do the following in the .rst

```diff
-    from stdio_mgr import stdio_mgr
+    >>> from stdio_mgr import stdio_mgr
```

Need to dig into this again and find out why.

Reply:
That is weird. If I checkout your branch and comment out the `add_stdio_mgr` fixture in `conftest.py`, all of the code blocks still pass. I had thought that each code block was executed in a clean namespace, but I guess not.

{fiddles a bit}

It looks like code samples that are just bare python:
```
from stdio_mgr import stdio_mgr
```
may be executed in separate, clean namespaces, whereas code blocks that're REPL-like:
```
>>> from stdio_mgr import stdio_mgr
```
are executed in a common namespace.

-----

If I comment out the import in the first block in README.rst:
```
>>> # from stdio_mgr_import stdio_mgr
```
and also comment out the `add_stdio_mgr` fixture, the README doctests fail. But, if I uncomment the `add_stdio_mgr` fixture, the README doctests pass.

Reply:
Hm. No, that's not it -- removing the `>>>` and `...` from the `stderr` example with `add_stdio_mgr` commented out does **not** cause the doctest to fail. 

I'm not sure exactly what's going on.

Reply:
Regardless, removing `>>> import os` and adding an appropriate import and `doctest_namespace` assignment in `conftest.py` does seem to work for me.

Reply:
Yup; that worked